test/Microsoft.AspNet.Mvc.ModelBinding.Test/Validation/DataAnnotationsModelValidatorProviderTest.cs

```diff
@@ -172,41 +178,15 @@ public void IValidatableObjectGetsAValidator()
             var mockValidatable = new Mock<IValidatableObject>();
             var metadata = _metadataProvider.GetMetadataForType(mockValidatable.Object.GetType());
 
-            // Act
-            var validators = provider.GetValidators(metadata);
-
-            // Assert
-            Assert.Single(validators);
-        }
-#endif
-
-        // Integration with metadata system
-
-        [Fact]
-        public void DoesNotReadPropertyValue()
```

Comment:
Correct. It's now **impossible** for this to happen. Nothing in this code path has access to the model. It's actually been this way since the first MM change I made.